settings.gradle.kts

```diff
@@ -2,8 +2,10 @@ rootProject.name = "FastAsyncWorldEdit"
 
 include("worldedit-libs")
 
-listOf("bukkit", "core").forEach {
+listOf("bukkit", "core", "fabric").forEach {
     include("worldedit-libs:$it")
     include("worldedit-$it")
 }
 include("worldedit-libs:core:ap")
+
+include("worldedit-core:doctools")
```

Comment:
Why are the doctools included? We don't build the module.

Reply:
No reason, I just copied some stuff  from upstream to see if it would fix, might've copied that as well by accident